up42/asset.py

```diff
@@ -103,14 +103,24 @@ def update_metadata(
         Update the metadata of the asset.
 
         Args:
-            title: The title string to be assigned to the asset.
-            tags: A list of tag strings to be assigned to the asset.
+            title: The title string to be assigned to the asset. No value will keep the existing title.
```

Comment:
Maybe we can improve this part.
I am not sure what do we mean by `"No value will keep the existing title."`
Same for the next line.
@dustgalactic any suggestions?

Reply:
It means that if the user does not provide a title or tag it will no update the field. In the previous case (that it was the actual bug we are trying to solve) it removed title/tags.

Reply:
Changes to these files won't update the reference: https://sdk.up42.com/reference/asset-reference/#update_metadata

What we have on the reference right now:

* title: An editable asset title.
* tags: An editable list of tags to categorize the asset.

These arguments are optional. An "optionality" in an argument behavior means that nothing will be changed. My question is, why do you think we need to update these texts?

Reply:
Because there is an error that is fixed as we can't change the signature. Optional and = None means that the value is not optional anymore. It means that when the user does not set the `title` the value is set to None by default. The correct behavior is that the `title` is not set to any value.

The unwanted behavior that we have right now is that when the user does not set `title` and `title` is set to None, then the update metadata removes the asset title. The behavior we want is that when the user does not set `tilte` value and the `title` is set to None, then the update metadata ignores this field and keep the title as it is.

Reply:
Ok, will update the reference in this PR.